### ISSUE TITLE
Disappearing keystrokes in the address bar

### ISSUE THREAD

**Issue by jpr48** [brave/browser-laptop#2197]
**Did you search for similar issues before submitting this one?**
Yup
**Describe the issue you encountered:**
Start typing in the address bar, and about half the time, they all will disappear after a second or two, and you will have to start over.
**Expected behavior:**
Not disappearing!
- Platform (Win7, 8, 10? OS X? Linux distro?):
Win 8.1 64bit
- Brave Version:
0.10.3
- Steps to reproduce:
    1. Just start typing and watch
    2.
    3.

- Screenshot if needed:

- Any related issues:

**Comment by PatrickRyanLane** [comment]
I've seen this issue as well. 

It seems to happen when users open a new tab using either the + button or ctrl+T shortcut. The initial response is a tab that opens displaying **about:newtab** in the tab bar. That tab refreshes after about half a second to an actual new tab. 

In that half second users have the ability to start typing a few letters, which then get wiped out on the reload from **about:newtab** to that actual new tab.

I hope this extra info will help fix the issue. Thanks Brave Team!

**Comment by semaj** [comment]
This issue is huge for me - I'd say most devs are CMD-T'ing all the time, and it's very annoying to type the first part of an address just to watch it get wiped out a split-second later.

**Comment by PatrickRyanLane** [comment]
Just adding a bit more info here. I believe this issue is related to [#847](https://github.com/brave/browser-laptop/issues/847) which is closed, but I think remains unsolved or was broken by another merge.

**Comment by stianjensen** [comment]
Seems to me that it's fixed in 0.10.4

**Comment by bbondy** [comment]
Can someone else confirmed fixed in 0.10.4? If so I'll close this.

**Comment by PatrickRyanLane** [comment]
@bbondy looks to me like it is fixed! Thanks everyone.

**Comment by bbondy** [comment]
Closing since 2 people confirmed it was fixed. Not sure what fixed this but we can re-open if it comes back or someone else still has it.

**Comment by bbondy** [comment]
likely something with autocomplete changes fixed.

**Comment by PatrickRyanLane** [comment]
This issue looks like it is cropping up again. Being reported in issue #2812

**Comment by bbondy** [comment]
we have a change in 0.11.4 which I think will re-fix, please open after that release this week if you see it.

**Comment by PatrickRyanLane** [comment]
@bbondy I just updated to 0.11.4 and the issue has evolved. If I try to open a new tab when I am already on a website, the new tab page will still have the issue and refresh about .5 secs after it opens, deleting all of my text. If I already have a blank new tab open and try to open another new tab, the new tab will not have the issue and will not refresh. 

**Issue e.g.** I have github.com and gizmodo.com open in two tabs -> I open a new blank tab -> begin typing into address bar -> new tab refreshes and I lose the text I've typed

**Non-issue e.g.** I have github.com and a blank new tab page open -> I open a new blank tab -> begin typing into address bar -> new tab does not refresh and my text stays
